### PR TITLE
Sohag project

### PR DESCRIPTION
**Pull request by SohagHossain9983**
homepag, cardpage, checkoutpage

### CODE REVIEW

File: components/bannar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Image from "next/image"
+import Button from "../global/button"
+import BannarImg from "../../assets/img/poster-03 1.png"
+
+const Bannar = () =>{
+    return(
+        <div className="" id="homeBannar">
```
Comment (naimuddin374) [inline components/bannar/index.jsx]: remove class name if has empty class

File: components/best-products/index.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import Image from "next/image"
+import BestProductsImgA from "../../assets/img/product-39 1.png"
+import BestProductsImgB from "../../assets/img/product-38.png"
+import BestProductsImgC from "../../assets/img/laptop (1).png"
+import StarReating from "../global/starReating"
+import Heading from '../global/headingTitle'
+
+const BestProducts =()=>{
+    return(
+        <div>
```
Comment (naimuddin374) [inline components/best-products/index.jsx]: remove empty div if has not needed

File: components/chaeck-out-card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                    <input type="radio" id="charg" name="shipping" value="HTML"/>
+                    <label for="charg">Shipping Free</label>
+                </div>
+                <div>
+                    <input type="radio" id="charg" name="shipping" value="HTML"/>
+                    <label for="charg">Shipping Fee $20</label>
+                </div>
+           </div>
+           <hr/>
+           <div className="d-flex justify-content-between px-4  ">
```
Comment (naimuddin374) [inline components/chaeck-out-card/index.jsx:34]: Remove empty class space

File: styles/components/best-products/index.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        color: white;
+        opacity: 0;
+        
+    }
+}
+#BestProducts:hover .buttonArea{
+    transition: all 0.4s;
+    opacity: 1;
+
+}
```
Comment (naimuddin374) [inline styles/components/best-products/index.scss]: remove commented code

File: styles/components/layout/_top-header.scss [hunk 5]
```diff
@@ -32,4 +32,11 @@
        margin: 0 10px;
     }
 
-}
\ No newline at end of file
+}
+
```
Comment (naimuddin374) [inline styles/components/layout/_top-header.scss]: remove

**Review by naimuddin374 — CHANGES_REQUESTED** [review]
fix these things

**Review by rahmanshifur — CHANGES_REQUESTED** [review]
hear new changes are unnecessary div, class-name comments and sob codes.. "The changes are good"

**Review by naimuddin374 — APPROVED** [review]

**Review by naimuddin374 — APPROVED** [review]
good to go

**Review by rahmanshifur — APPROVED** [review]

**Pull request merged**